packages/backend/src/nest/storage/certificates/certificates.store.ts

```diff
@@ -97,6 +92,11 @@ export class CertificatesStore {
   public updateMetadata(metadata: CommunityMetadata) {
     if (!metadata) return
     this.metadata = metadata
+    // Community metadata is required for validating certificates, so
+    // we re-validate certificates once community metadata is set. It
+    // might also make sense to initialize CertificateStore after
+    // CommunityMetadata is received instead of doing this.
+    this.loadedCertificates()
```

Comment:
The previous wait mechanism with setTimeout appeared to be causing another JS timing issue where callbacks were run out of order and caused some certificates to not be sent to the frontend. The other changes in this PR seemed to flush it out into the open. So this change is fixing that. Now instead of waiting, we throw an error if community metadata is missing and revalidate when community metadata is available.